pocketimdb/movie_feedbacks/views.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import viewsets, mixins
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.status import HTTP_201_CREATED
+from .models import MovieFeedback, Movie
+from .serializers import AddMovieFeedbackSerializer
+
+class MovieFeedbackViewSet(mixins.CreateModelMixin,
+                viewsets.GenericViewSet):
+
+    queryset = MovieFeedback.objects.all()
+    permission_classes = [IsAuthenticated]
+    serializer_class = AddMovieFeedbackSerializer
+
+    def create(self, request):
+        data = request.data
+        data['user'] = request.user.id
+        serializer = self.get_serializer(data=data)
```

Comment:
ovo je lose, iz razloga sto dopustas da korisnik posalje preko request korisnika, iako ga neces na kraju iskorsiti stece se pogresan utisak o samom endpoint-u i na koji nacin funkcionise. Nema potreba na ovaj nacin da kreiramo ovaj model, mozes imati strukturu 

```
   get_serializer(data=data)

   serializer.is_valid(raise_exception=True)

   MovieFeedback.objects.create(....podaci koje treba da iskorisits kako bi kreirao record)

Reply:
Kada koristim create metodu nad modelom, tada mi trazi kompletan Movie objekat, a ja kroz request saljem id samo. A kada radim save pomocu serializer-a pozivom metode `perform_create`, dovoljno je da mu dam id da uradi save. Zbog toga mi je jednostavnije da radim tako. Ovako ili moram slati ceo Movie objekat (sto mi se nije cinilo ispravnim) ili da sam dovlacim iz baze Movie objekat na osnovu id-a. A serializer pri validaciji zahteva da to bude id nekog postojeceg Movie objekta, dakle ne bih ni mogao da saljem Movie objekat.